### PR TITLE
Notify Slack on new PRs

### PR DESCRIPTION
**Pull request by darwinfroese**

### CODE REVIEW

File: src/gitHandlers/pullRequest.ts [hunk 1]
```diff
@@ -36,7 +38,13 @@ export const pullRequest = async ({ action, pull_request }: IPullRequestEvent) =
...
             }
 
-            if (action === "opened" && ENABLE_PR_QUOTES && prQuotes.coolCats.includes(user.login)) {
+            if (action !== "opened")
+                break;
+
+            let PRUrl = `${GITHUB_URL}repos/${owner}/${name}/pulls/${number}`;
+            postMessage({ text: `@here A New PR is ready for review: ${PRUrl}`, channel: "hippo-devs" });
+
+            if (ENABLE_PR_QUOTES && prQuotes.coolCats.includes(user.login)) {
```
Comment (KevinCSimpson) [inline src/gitHandlers/pullRequest.ts:47]: Only came here to make sure this part didn't get lost.  Business critical.

File: src/api/slack/chat.ts [hunk 2]
```diff
@@ -8,5 +8,8 @@ export interface IPostMessageParams {
 
 export const postMessage = async (params: IPostMessageParams) => {
     const web = new WebClient(SLACK_TOKEN);
-    return await web.chat.postMessage(params);
+    return await web.chat.postMessage({
```
Comment (KevinCSimpson) [inline src/api/slack/chat.ts:11]: What's the reason for this change?

**Review by KevinCSimpson — APPROVED** [review]

File: src/api/slack/chat.ts (hunk 2 above)
Comment (darwinfroese) [reply src/api/slack/chat.ts:11]: No idea, vscode said that it was an error and couldn't do the conversion and the docs said to do it like that and it fixed the error :shrug:

File: src/gitHandlers/pullRequest.ts (hunk 1 above)
Comment (darwinfroese) [reply src/gitHandlers/pullRequest.ts:47]: Its critical. You have to keep it!

**Review by darcymiranda — APPROVED** [review]
